app/backend/src/clj/teet/file/file_export.clj

```diff
@@ -57,6 +57,31 @@
          (catch Throwable e
            (log/info e "io/copy failed in zip creation")))))))
 
+(defn- file-name-with-sequence-number [seq fn]
+  (let [{:keys [extension description]}
+        (filename-metadata/name->description-and-extension fn)
+        new-description (str description " (" seq ")")]
+
+    (if (not-empty extension)
+      (str new-description "." extension)
+      ;; else:
```

Comment:
I would refactor this ext check into the function with descriptive name to avoid ;; ;; comments